books/free-programming-books-ko.md

```diff
@@ -237,8 +239,15 @@
 
 ### Rust
 
+* [러스트 코딩인사이트](https://coding-insight.com/docs/rust/start/rust)
+* [러스트 투어](https://tourofrust.com/00_ko.html)
+* [러스트 프로그래밍 공식가이드](https://doc.rust-kr.org/title-page.html)
```

Comment:
Don't invent titles! Use the title given by the resource. "The Rust Programming Language"